### PR TITLE
return errors for invalid key pem blocks

### PR DESCRIPTION
**Pull request by eiais**
Signed-off-by: Kyle Spiers <kyle@spiers.me>

### CODE REVIEW

File: trustmanager/keys.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"sort"
 	"strings"
 
+	"fmt"
```
Comment (riyazdf) [inline trustmanager/keys.go]: super nit: let's move this to the block above with the other standard golang imports

Comment (eiais) [reply trustmanager/keys.go]: fixed

**Comment by riyazdf** [comment]
CI is failing with a `gofmt` error on `trustmanager/keys.go` - could you amend your commit with the reformatted file?

**Comment by riyazdf** [comment]
LGTM!

**Pull request merged**